src/vizier/services/cloud_connector/bridge/server.go

```diff
@@ -81,7 +81,7 @@ spec:
       serviceAccountName: pl-updater-service-account
       containers:
       - name: updater
-        image: gcr.io/pixie-oss/pixie-dev/vizier/vizier_updater_image:__VIZIER_UPDATER_IMAGE_TAG__
+        image: gcr.io/pixie-oss/pixie-prod/vizier-vizier_updater_image:__VIZIER_UPDATER_IMAGE_TAG__
```

Comment:
I think you'll also need to update `ParseJobYAML` in `cloud_connector/bridge/vzinfo.go`, since it tries to do some image path replacement there and assumes you have `/vizier` in the image path.